signbank/video/tests/test_views.py

```diff
@@ -196,3 +197,116 @@ def test_upload_glossvideo_with_custom_redirect(self):
         response = self.client.post(reverse('video:upload_glossvideo_gloss'), {
             'videofile': self.testfile, 'gloss': self.testgloss.pk, 'video_type': self.video_type.machine_value, 'redirect': reverse('video:manage_videos')})
         self.assertEqual(response.url, reverse('video:manage_videos'))
+
+class ExportGlossvideoCsvTestCase(TestCase):
+    def setUp(self):
+        # Create user and add permissions
+        self.user = User.objects.create_user(
+            username="test", email=None, password="test")
+        permission = Permission.objects.get(codename='export_csv')
```

Comment:
**suggestion**: because this isn't used later, you could avoid assigning it:

```python
self.user.user_permissions.add(
    Permission.objects.get(codename='export_csv')
)
```